config/runtime.exs

```diff
@@ -61,3 +62,10 @@ block_time_ms =
 
 config :lambda_ethereum_consensus, LambdaEthereumConsensus.Telemetry,
   block_processing_buckets: [0.5, 1.0, 1.5, 2, 4, 6, 8] |> Enum.map(&(&1 * block_time_ms))
+
+if is_nil(jwt_secret) do
+  Logger.warning(
```

Comment:
is Logger ready in runtime.ex?

Reply:
The warning works 🤔. Let me check if it's ready or not